integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -782,14 +783,15 @@ private void createLoadBalancer() throws Exception {
 
   private void callShellScriptByExecToPod(String username, String password, String webappName)
       throws Exception {
+    String asn = adminServerName.toLowerCase(Locale.ENGLISH).replace('_', '-').replace('.', '-');
```

Comment:
I think you can replace the characters '_ 'and '.' with '-' for adminServerName and managedServerNameBase  in the map itself in initialize() method after loading the yaml to map.